py-polars/polars/dataframe/frame.py

```diff
@@ -7355,18 +7358,21 @@ def get_columns(self) -> list[Series]:
         """
         return [wrap_s(s) for s in self._df.get_columns()]
 
-    def get_column(self, name: str) -> Series:
+    def get_column(self, name: str, *, default: Any | NoDefault = no_default) -> Series:
```

Comment:
I think we should improve return `Series | NoDefault`? Or maybe `Any`?. And we then probably need an @overload

Reply:
Done; if "default" is not set (or is a Series), we maintain the `Series` return type; if "default" is set to anything else, the return type is `Any`.